molgenis-data/src/main/java/org/molgenis/data/transaction/TransactionConfig.java

```diff
@@ -16,10 +17,14 @@
 	@Autowired
 	private MolgenisTransactionManager transactionManager;
 
+	// This cannot be a app setting because apps settings are not loaded yet when we use it
+	@Value("${use.transaction.log:true}")
```

Comment:
Why have an option to disable transaction log? Transaction log is required in order for the rollback mechanism to work, right? What problem were you trying to solve?